docs/development/priority-classes.md

```diff
@@ -23,7 +23,7 @@ When using the `gardener-operator` for managing the garden runtime and virtual c
 |---------------------------------- |-----------|----------------------------------------------------------------------------------------------------------------|
 | `gardener-garden-system-critical` | 999999550 | `gardener-operator`, `gardener-resource-manager`, `istio`                                                      |
 | `gardener-garden-system-500`      | 999999500 | `virtual-garden-etcd-events`, `virtual-garden-etcd-main`, `virtual-garden-kube-apiserver`, `gardener-apiserver`|
-| `gardener-garden-system-400`      | 999999400 | `virtual-garden-gardener-resource-manager`, `gardener-admission-controller`                                    |
+| `gardener-garden-system-400`      | 999999400 | `virtual-garden-gardener-resource-manager`, `gardener-admission-controller`, `gardener-scheduler`              |
```

Comment:
I think it should go at least one level lower - `300` should be good enough. Maybe `200` is also appropriate compared to the criticality of the other components in the `300` level. WDYT?

Reply:
okay 😄